### PR TITLE
Add cert extensions on certificate utilty

### PR DESCRIPTION
**Pull request by acdemyers**
Add certificate extension flag on createSSLCertificate utility.

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops [hunk 1]
```diff
@@ -184,6 +184,13 @@ sslCert.option-desc.keyType=\
 \tfrequently the format provided by certificate authorities when\n\
 \tissuing certificates.
 
+sslCert.option-key.ext=\
+\ \ \ \ --extInfo=certificate extension information
+sslCert.option-desc.ext=\
+\tCertificate extension information to add to the certificate.\n\
+\tThe default extension will be a Subject Alternative Name entry using\n\
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops]: Recommend changing:
The default extension will be a Subject Alternative Name entry using
to
The default extension is a Subject Alternative Name entry using

Comment (acdemyers) [reply dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops]: Thank you @jantley-ibm!   I've made the suggested change.

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Pull request merged**